src/porepy/numerics/ad/operators.py

```diff
@@ -575,7 +588,11 @@ def __radd__(self, other):
         return self.__add__(other)
 
     def __rsub__(self, other):
-        return self.__sub__(other)
```

Comment:
Would other.__sub__(self) work as an alternative fix?

Reply:
This depends on what the other is and if it supports that syntax.
My proposal makes that question obsolete since other is wrapped into the AD framework using `_parse_other`, which tells us what is reasonable, i.e. what it supports.